dashboard/app/models/peer_review.rb

```diff
@@ -41,6 +41,40 @@ class PeerReview < ActiveRecord::Base
     escalated: 2
   }
 
+  def self.pull_review_from_pool(script, user)
+    # Find the first review such that meets these criteria
+    # Review is for this script
+    # I am not the submitter X
+    # Review status is nil
+    # Reviewer is nil or it has been assigned for more than a day
+    # Reviewer is not currently reviewing this level source
+    level_sources_reviewing = PeerReview.where(reviewer: user, script: script).pluck(:level_source_id)
+
+    peer_review = where(script: script).
+        where.not(submitter: user).
+        where(status: nil).
+        where('reviewer_id is null or created_at < NOW() - 1').
```

Comment:
I don't think this actually subtracts 1 day:
```
mysql> select date(now()), date(now() - 1);
+-------------+-----------------+
| date(now()) | date(now() - 1) |
+-------------+-----------------+
| 2016-06-22  | 2016-06-22      |
+-------------+-----------------+
```
Clearer to write `now() - interval 1 day`?